src/Nest/Document/Single/Get/GetResponse.cs

```diff
@@ -1,13 +1,14 @@
-﻿using System;
-using System.Runtime.Serialization;
+﻿using System.Runtime.Serialization;
 using Elasticsearch.Net;
 
 namespace Nest
 {
+	// TODO: Looks like this can be removed?

```

Comment:
TODO?

Reply:
Related to the removal of response interfaces that @Mpdreamz was working on.

Added to discuss before GA, but not needed to address to merge this PR.